spec/requests/pages_spec.rb

```diff
@@ -56,17 +101,17 @@
       it 'creats a petitionform' do
         expect_any_instance_of( ActionKitConnector::Client ).to(
           receive(:create_petitionform).
-            with(hash_including(
-              {
-                :client_hosted=>true,
-                :client_url=>"http://example.com",
-                :ask_text=>"Dummy ask",
-                :thank_you_text=>"Dummy thank you",
-                :statement_text=>"Dummy statement",
-                :page=>"https://act.sumofus.org/rest/v1/petitionpage/12574/"
-              }
-            )
+          with(hash_including(
+            {
+              :client_hosted=>true,
+              :client_url=>"http://example.com",
+              :ask_text=>"Dummy ask",
+              :thank_you_text=>"Dummy thank you",
+              :statement_text=>"Dummy statement",
+              :page=>"https://act.sumofus.org/rest/v1/petitionpage/12574/"
+            }
           )
+              )
```

Comment:
I don't understand - it looks like you removed a bunch of indentation but it makes the formatting look kinda worse.
```
            }
          )
              )
        )
```
is pretty wonky looking